guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,20 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+
+.Prerequisites for Disconnected Environments
+If you run {Project} in a disconnected environment, ensure it meets the following prerequisites:
+
+* You must obtain and deploy Leapp metadata manually.
+For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You require access to {RHEL} and {Project} packages.
+* Obtain the ISO files for {RHEL} 8 and {Project}.
+For more information, see xref:upgrading_a_disconnected_satellite[].
+* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
 endif::[]
+ifndef::satellite[]
 * Access to available repositories or a local mirror of repositories.
+endif::[]
 * If you previously upgraded {Project} from an earlier version, and the `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL, empty `/var/lib/pgsql` before proceeding.
 * During the upgrade, the PostgreSQL data is moved from `/var/opt/rh/rh-postgresql12/lib/pgsql/data/` to `/var/lib/pgsql/data/`.
 If these two paths reside on the same partition, no further action is required.
```

Comment:
These are now rendered as part of the "Prerequisites for Disconnected Environments" for Satellite, but they do apply for both!

Reply:
It's rendered in Foreman as well.